stablehlo/tests/verify_reduce.mlir

```diff
@@ -116,27 +118,10 @@ func.func @reduce_odd_num_args(%arg0: tensor<?x?xf32>, %arg1: tensor<?x?xf32>,
 
 // -----
 
-func.func @reduce_zero_args(%arg0: tensor<?x?xf32>, %arg1 : tensor<f32>)
-    -> (tensor<?xf32>) {
-
-  // expected-error@+1 {{expects the size of operands to be even and >= 2}}
```

Comment:
Same question - why is this test point deleted

Reply:
Yeah, same reply as the other comment.